hotness/consumers.py

```diff
@@ -297,8 +351,8 @@ def handle_buildsys_scratch(self, msg):
         text2 = "%s %s" % (subtitle, url)
 
         # Followup on bugs we filed
-        if task_id in self.triggered_task_ids:
-            bugs.append((self.triggered_task_ids.pop(task_id), text1))
+        #if task_id in self.triggered_task_ids:
+        #    bugs.append((self.triggered_task_ids.pop(task_id), text1))
```

Comment:
Well, 
I can return from rebase-helper task_id but for this case
I need to download all rpm packages for comparing packages by rpmdiff, abipkgdiff, pkgdiff, etc.

Is it possible to do it?
Or better say how can I download packages which are done by fedpkg.
I will call rebase-helper function for comparing.